examples/spec/integration/schedule_child_workflow_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'workflows/schedule_child_workflow'
+require 'workflows/hello_world_workflow'
+
+describe ScheduleChildWorkflow, :integration do
+  let(:cron_schedule) { "*/6 * * * *" }
+
+  it 'schedules a child workflow with a given cron schedule' do
```

Comment:
We might also need to clean up the scheduled workflow after running this spec

Reply:
It should be getting cleaned up currently, with the combination of the parent close policy and the execution timeout of the parent. Do you think we should further ensure the child is terminated?